### PR TITLE
Remove double slashes in Strip control codes challenge

### LINKED ISSUE #38436: Rosetta Code-Strip Control Codes: Assertions Assert Incorrect Strings
The testing assertion argument strings don't match the input strings. They're missing second slashes. Therefore the test won't accept correct answers:

image

Test Page: [Strip Control Codes](https://www.freecodecamp.org/learn/coding-interview-prep/rosetta-code/strip-control-codes-and-extended-characters-from-a-string)
Assertions found here: [Strip Control Codes](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/curriculum/challenges/english/08-coding-interview-prep/rosetta-code/strip-control-codes-and-extended-characters-from-a-string.english.md)

### PR DESCRIPTION
**Pull request by agatapst**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #38436

### CODE REVIEW

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/strip-control-codes-and-extended-characters-from-a-string.english.md [hunk 1]
```diff
@@ -27,17 +27,17 @@ tests:
   - text: <code>strip</code> should be a function.
     testString: assert(typeof strip == 'function', '<code>strip</code> should be a function.');
```
Comment (RandellDawson) [inline curriculum/challenges/english/08-coding-interview-prep/rosetta-code/strip-control-codes-and-extended-characters-from-a-string.english.md]:
```suggestion
    testString: assert(typeof strip == 'function');
```

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/strip-control-codes-and-extended-characters-from-a-string.english.md [hunk 2]
```diff
@@ -27,17 +27,17 @@ tests:
...
   - text: <code>strip("abc")</code> should return a string.
-    testString: assert(typeof strip("\ba\\x00b\n\rc\fd\xc3") == 'string', '<code>strip("abc")</code> should return a string.');
-  - text: <code>strip("\\ba\\x00b\\n\\rc\\fd\\xc3")</code> should return <code>"abcd"</code>.
-    testString: assert.equal(strip("\ba\x00b\n\rc\fd\xc3"), "abcd", '<code>strip("\\ba\\x00b\\n\\rc\\fd\\xc3")</code> should return <code>"abcd"</code>.');
-  - text: <code>strip("\\u0000\\n abc\\u00E9def\\u007F")</code> should return <code>" abcdef"</code>.
-    testString: assert.equal(strip("\u0000\n abc\u00E9def\u007F"), " abcdef", '<code>strip("\\u0000\\n abc\\u00E9def\\u007F")</code> should return <code>" abcdef"</code>.');
-  - text: <code>strip("a\\n\\tb\\u2102d\\u2147f")</code> should return <code>"abdf"</code>.
-    testString: assert.equal(strip("a\n\tb\u2102d\u2147f"), "abdf", '<code>strip("a\\n\\tb\\u2102d\\u2147f")</code> should return <code>"abdf"</code>.');
+    testString: assert(typeof strip("abc") == 'string', '<code>strip("abc")</code> should return a string.');
```
Comment (RandellDawson) [inline curriculum/challenges/english/08-coding-interview-prep/rosetta-code/strip-control-codes-and-extended-characters-from-a-string.english.md]:
```suggestion
    testString: assert(typeof strip("abc") == 'string');
```

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/strip-control-codes-and-extended-characters-from-a-string.english.md [hunk 3]
```diff
@@ -27,17 +27,17 @@ tests:
...
+  - text: <code>strip("\ba\x00b\n\rc\fd\xc3")</code> should return <code>"abcd"</code>.
+    testString: assert.equal(strip("\ba\x00b\n\rc\fd\xc3"), "abcd", '<code>strip("\ba\x00b\n\rc\fd\xc3")</code> should return <code>"abcd"</code>.');
```
Comment (RandellDawson) [inline curriculum/challenges/english/08-coding-interview-prep/rosetta-code/strip-control-codes-and-extended-characters-from-a-string.english.md]:
```suggestion
    testString: assert.equal(strip("\ba\x00b\n\rc\fd\xc3"), "abcd");
```

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/strip-control-codes-and-extended-characters-from-a-string.english.md [hunk 4]
```diff
@@ -27,17 +27,17 @@ tests:
...
+  - text: <code>strip("\u0000\n abc\u00E9def\u007F")</code> should return <code>" abcdef"</code>.
+    testString: assert.equal(strip("\u0000\n abc\u00E9def\u007F"), " abcdef", '<code>strip("\u0000\n abc\u00E9def\u007F")</code> should return <code>" abcdef"</code>.');
```
Comment (RandellDawson) [inline curriculum/challenges/english/08-coding-interview-prep/rosetta-code/strip-control-codes-and-extended-characters-from-a-string.english.md]:
```suggestion
    testString: assert.equal(strip("\u0000\n abc\u00E9def\u007F"), " abcdef");
```

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/strip-control-codes-and-extended-characters-from-a-string.english.md [hunk 5]
```diff
@@ -27,17 +27,17 @@ tests:
...
+  - text: <code>strip("a\n\tb\u2102d\u2147f")</code> should return <code>"abdf"</code>.
+    testString: assert.equal(strip("a\n\tb\u2102d\u2147f"), "abdf", '<code>strip("a\n\tb\u2102d\u2147f")</code> should return <code>"abdf"</code>.');
```
Comment (RandellDawson) [inline curriculum/challenges/english/08-coding-interview-prep/rosetta-code/strip-control-codes-and-extended-characters-from-a-string.english.md]:
```suggestion
    testString: assert.equal(strip("a\n\tb\u2102d\u2147f"), "abdf");
```

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/strip-control-codes-and-extended-characters-from-a-string.english.md [hunk 6]
```diff
@@ -27,17 +27,17 @@ tests:
...
   - text: <code>strip("Français.")</code> should return <code>"Franais."</code>.
     testString: assert.equal(strip("Français."), "Franais.", '<code>strip("Français.")</code> should return <code>"Franais."</code>.');
```
Comment (RandellDawson) [inline curriculum/challenges/english/08-coding-interview-prep/rosetta-code/strip-control-codes-and-extended-characters-from-a-string.english.md]:
```suggestion
    testString: assert.equal(strip("Français."), "Franais.");
```

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/strip-control-codes-and-extended-characters-from-a-string.english.md [hunk 7]
```diff
@@ -27,17 +27,17 @@ tests:
...
-  - text: <code>strip("123\\tabc\\u0007DEF\\u007F+-*/€æŧðłþ")</code> should return <code>"123abcDEF+-*/"</code>.
-    testString: assert.equal(strip("123\tabc\u0007DEF\u007F+-*/€æŧðłþ"), "123abcDEF+-*/", '<code>strip("123\\tabc\\u0007DEF\\u007F+-*/€æŧðłþ")</code> should return <code>"123abcDEF+-*/"</code>.');
+  - text: <code>strip("123\tabc\u0007DEF\u007F+-*/€æŧðłþ")</code> should return <code>"123abcDEF+-*/"</code>.
+    testString: assert.equal(strip("123\tabc\u0007DEF\u007F+-*/€æŧðłþ"), "123abcDEF+-*/", '<code>strip("123\tabc\u0007DEF\u007F+-*/€æŧðłþ")</code> should return <code>"123abcDEF+-*/"</code>.');
```
Comment (RandellDawson) [inline curriculum/challenges/english/08-coding-interview-prep/rosetta-code/strip-control-codes-and-extended-characters-from-a-string.english.md]:
```suggestion
    testString: assert.equal(strip("123\tabc\u0007DEF\u007F+-*/€æŧðłþ"), "123abcDEF+-*/");
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@agatapst Since most of the `testString`s have been modified, can you please go ahead and remove the unnecessary assert message arguments?  You can click `Commit suggestion` for each and a commit will be made.

**Comment by agatapst** [comment]
@RandellDawson Yes, thank you! 😊

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Review by RandellDawson — APPROVED** [review]

**Comment by ojeytonwilliams** [comment]
@agatapst Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Pull request merged**